odoo/sql_db.py

```diff
@@ -309,6 +327,19 @@ def execute(self, query, params=None, log_exceptions=None):
         try:
             params = params or None
             res = self._obj.execute(query, params)
+
+            if (getattr(threading.currentThread(), 'testing', False) or
+                config.get('test_enable') or not (QUERY_CATCH_VERBOSE and
+                (bytes(QUERY_TO_CATCH, encoding='UTF-8') in self._obj.query))):
```

Comment:
What about using the same style to convert string to bytes?

```suggestion
                (QUERY_TO_CATCH.encode('UTF-8') in self._obj.query))):
```

Reply:
Done